docs/docs/recoil-sync/api/RecoilURLSyncTransit.md

```diff
@@ -7,7 +7,7 @@ A component from the [Recoil Sync library](/docs/recoil-sync/introduction) to sy
 
 This is identical to the [`<RecoilURLSync>`](/docs/recoil-sync/api/RecoilURLSync) component except that it provides built-in [Transit encoding](https://github.com/cognitect/transit-js).
 
-```jsx
+```tsx
```

Comment:
Did you notice a problem with `jsx` needing to switch to `tsx`?

Reply:
It's probably a good idea to switch all the example/documentation syntax from Flow to TypeScript, though I do worry a tad about consistency...

Reply:
ah yes. with JSX, prettier seemed to mangle the generic it doesnt understand (i'm editing the MDs within vscode if that matters):

```

const ViewState =
  atom <
  ViewState >
  {
    key: 'ViewState',
    default: new ViewState(true, [1, 2]),
    effects: [syncEffect({storeKey: 'transit-url', refine: viewStateChecker})],
  };
```

Reply:
My VSCode is a mess editing the docs...  Check out how it actually looks with `/docs/yarn start`

Reply:
Hmm what do you see?   Seems good on my local:

<img width="1415" alt="image" src="https://user-images.githubusercontent.com/106260338/192068029-c1574808-2598-4cba-8778-aa87de1a48be.png">

Reply:
Oh, I just meant in general I can't really rely on VSCode for prettier or linting the docs, so I edit it there but rely on `/docs/yarn start` to see how it actually looks.